lib/index.js

```diff
@@ -22,11 +22,15 @@ webFrame.setZoomLevelLimits(1, 1);
 
 const store_ = createStore(
   rootReducer,
-  applyMiddleware(
-    thunk,
-    plugins.middleware,
-    thunk,
-    effects
+  compose(
+    applyMiddleware(
+      thunk,
+      plugins.middleware,
+      thunk,
+      effects
+    ),
+    // Redux DevTools extension integration
+    window.devToolsExtension ? window.devToolsExtension() : f => f
```

Comment:
Just implemented redux-devtools yesterday in another project. since it's a middleware what I was saying makes no sense at all haha.